### PR TITLE
[W5.10][W11-A2] Kevin Leonardo

### PR DESCRIPTION
**Pull request by k-l-a**
Pull Request for LO W5.10 SRP
Split TextUi class into Formatter.
Formatter now functions to format the messages to be displayed.
TextUi now uses a Formatter to format messages.
Removed redundant methods and variables from TextUi.
Implementation is such that only TextUi class is changed from existing codebase.

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                storageFileInfo,
+                DIVIDER);
+    }
+
+    public void showGoodbyeMessage(String goodbyeMessage) {
+        showToUser(goodbyeMessage, DIVIDER, DIVIDER);
+    }
+
+
+    public void showInitFailedMessage(String failMessage) {
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java]: I guess you have mistaken the exercise?
The Formatter class should do exclusive formatting and UI only the interaction with the user.
The current implementation moved stuff from UI to Formatter, and Formatter is doing more than just formatting (both formatting and showing to user), thus violating SRP

**Comment by okkhoy** [comment]
@k-l-a 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/ui/Formatter.java (hunk 1 above)
Comment (k-l-a) [reply src/seedu/addressbook/ui/Formatter.java]: Ah yes I seem to be mistaken. Formatter should only format and return the formatted string which the UI then shows that string to the use.

**Comment by k-l-a** [comment]
Revised

**Pull request closed without merging**